sys/mips/mips/support.S

```diff
@@ -114,7 +107,7 @@
 
 	.text
 
-#ifdef CHERI_IMPLICIT_USER_DDC
+#ifndef CPU_CHERI
```

Comment:
I kept CPU_CHERI here only because the rest of the file used it.  In general we tend to use `__has_feature()` everywhere else though.  I don't remember if I used CPU_CHERI at all in any of the RISC-V code.